src/components/PokemonCard/PokemonCard.jsx

```diff
@@ -0,0 +1,22 @@
+import './PokemonCard.css';
+
+    function PokemonCard({ pokemon }) {
+        return (
+            <div className="card">
+                <h3>{pokemon.name}</h3>
+                <img src={pokemon.image} alt={pokemon.name} />
+                <p><strong>Moves:</strong> {pokemon.moves}</p>
+                <p><strong>Weight:</strong> {pokemon.weight}</p>
+                <div className="abilities">
+                    <strong>Abilities:</strong>
+                    <ul>
+                        {pokemon.abilities.map((ability) => (
+                            <li key={pokemon.id}>{ability}</li>
+                        ))}
+                    </ul>
+                </div>
+            </div>
```

Comment:
In App.jsx staat dit component ook al in een div waardoor je dus een div in een div hebt. Het is netter om hier met semantische elementen te werken.